pubnub-memberships/src/integrationTest/kotlin/com/pubnub/entities/MembershipIntegTest.kt

```diff
@@ -65,7 +65,7 @@ class MembershipIntegTest {
         createSpace(SPACE_ID)
         val userIdWithCustomList: List<UserIdWithCustom> = listOf(UserIdWithCustom(USER_ID, MEMBERSHIP_CUSTOM), UserIdWithCustom(USER_ID_02, MEMBERSHIP_CUSTOM))
 
-        val addMembershipsResult = pubnub.addMembershipsOfSpace(spaceId = SPACE_ID, userIdWithCustomList = userIdWithCustomList).sync()
+        val addMembershipsResult = pubnub.addMembershipsOfSpace(spaceId = SPACE_ID, userIdsWithCustoms = userIdWithCustomList).sync()
```

Comment:
`userIdsWithCustoms` looks better for me, can we change the `val userIdWithCustomList` too?